packages/playwright-core/src/utils/multimap.ts

```diff
@@ -38,6 +38,16 @@ export class MultiMap<K, V> {
     return this._map.has(key);
   }
 
+  delete(key: K, value: V) {
+    const values = this._map.get(key);
+    if (!values)
+      return;
+    const index = values.indexOf(value);
+    if (index !== -1)
+      values.splice(index, 1);
```

Comment:
I wonder why `values` is not a set...

Reply:
I guess we did not want to clash the values. I can theoretically delete all matching.